portugues/revisa/tenta-consegue/sem-resposta/contato-desenvolvedor.md

```diff
@@ -1,2 +1,5 @@
 Voc� pergunta para o desenvolvedor quando a nova versao do aplicativo estar� dispon�vel para download.
 Por�m ele responde que nao est� mais envolvido com o mesmo.
+
+[A ideia do coco parecia �tima mesmo](coco/muito-quente.md)
+[Voc� vai atr�s do novo desenvolvedor](procura-novo-desenvolvedor/novo-desenvolvedor.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.